README.md

```diff
@@ -27,7 +27,12 @@ Github Actions currently builds a Windows x64 `.dll`, A Linux x64 `.so`, and a m
 You need:
 * git
 * Dart 3+
-* C++ build tools for your platform (Visual Studio, XCode, gcc, etc)
+* C++ build tools for your platform (Visual Studio, XCode, gcc, etc) 
+* For Windows 
+  * 2019 16.61 with 10.0.20348.0 SDK don't forget install Debugger Tools
+  * 2022 17 with ? SDK don't forget install Debugger Tools
```

Comment:
I think the windows SDK is always going to be the same, so I think the answer is VS 2019+ with Windows SDK 10.0.20348.0